armi/nucDirectory/tests/test_nuclideBases.py

```diff
@@ -420,13 +420,41 @@ def test_loadMcc3Data(self):
                 [nuc for nuc in data.keys() if data[nuc]["ENDF/B-VII.0"] is not None]
             )
 
-        for nuc, nb in nuclideBases.byMcc3Id.items():
+        for nuc, nb in nuclideBases.byMcc3IdEndfbVII0.items():
+            self.assertIn(nb.name, expectedNuclides)
+            self.assertEqual(nb.getMcc3IdEndfbVII0(), nb.mcc3idEndfbVII0)
+            self.assertEqual(nb.getMcc3IdEndfbVII0(), nuc)
+
+        # Subtract 1 nuclide due to DUMP2.
+        self.assertEqual(len(nuclideBases.byMcc3IdEndfbVII0), len(expectedNuclides) - 1)
+
+    def test_loadMcc3EndfVII1Data(self):
+        """Tests consistency with the `mcc-nuclides.yaml` input and the VII.1 nuclides in the data model.
+
+        .. test:: Test that MCC v3 IDs can be queried by nuclides.
+            :id: T_ARMI_ND_ISOTOPES4
```

Comment:
Again, check for the uniqueness of these test IDs.

Reply:
Done! Thank you.